Firestore/core/src/firebase/firestore/local/local_view_changes.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_LOCAL_LOCAL_VIEW_CHANGES_H_
+#define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_LOCAL_LOCAL_VIEW_CHANGES_H_
+
+#include <utility>
+
+#include "Firestore/core/src/firebase/firestore/core/view_snapshot.h"
+#include "Firestore/core/src/firebase/firestore/model/document_key_set.h"
+#include "Firestore/core/src/firebase/firestore/model/types.h"
+
+namespace firebase {
+namespace firestore {
+namespace local {
+
+/**
+ * Represents changes applying to the local view of a given query, including
+ * what documents are currently in view and out of view.
+ *
+ * Example usage: these changes are sent to the LocalStore by the View (via the
+ * SyncEngine) and are used to pin / unpin documents as appropriate.
+ */
+class LocalViewChanges {
+ public:
+  static LocalViewChanges MakeChanges(const core::ViewSnapshot& snapshot,
```

Comment:
This method is called `fromViewSnapshot` on Android, which seems like a more descriptive name. In the Web client, it's `fromSnapshot`. I'd suggest `FromViewSnapshot`. In general, it's a good idea to be consistent with other platforms wherever practical.

Reply:
Done. Will remember to check other platforms..